patching/plugin-commands-patching/src/stateFile.ts

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs'
+import path from 'path'
+import util from 'util'
+
+export type StateKey = string
```

Comment:
The type names are too generic. What is state key?

Reply:
https://github.com/pnpm/pnpm/commit/dd27764dd823fa6c77e81d1b9545ada839180ae3